General-Installing-Instructions.md

```diff
@@ -166,6 +166,12 @@ MySQL> GRANT SELECT ON mysql.time_zone_name TO cactiuser@localhost IDENTIFIED BY
 MySQL> flush privileges;
 ```
 
+Note that if your `root` (or equivalent) user does not have `SUPER` permissions, it may still be possible to `GRANT SELECT` privileges to the Cacti user via an `INSERT INTO mysql.tables_priv`.
```

Comment:
This line seems to be too long to me, not sure why the MDL check didn't raise it but it may be due to the inline code blocks.

Reply:
I presume you meant the next line (the SQL query) was too long, so I've fixed that. Other lines of documentation are just as long as this line but I can chop it down as well if you'd like.

Reply:
It was actually the line above the code example, most lines should be truncated to around 80 characters.  But worry not, I'll merge it and fix it up.